lua/nvim-tree/actions/node/open-file.lua

```diff
@@ -370,36 +370,35 @@ end
 ---@param mode string
 ---@param filename string
 function M.fn(mode, filename)
-  local fname = utils.escape_special_chars(filename)
```

Comment:
This is a change in behaviour that will affect linux and macos users.

Please put this behind the windows feature flag, to ensure that it only applies to windows users.

Something like
```
local fname
if utils.is_windows then
  fname = filename
else
  fname = utils.escape_special_chars(filename)
end
```

Reply:
Hi @alex-courtis , change of this line is caused by the revert of previous fix. I think there wasn't such line at the begining.

Reply:
Yes, I reverted all changes of #2903 . 

In #2903 `util.escape_special_chats` is called here and impact all file names below, which are not correct. (my tests failed)

Now and before #2903  `utils.escape_special_chars` is called in the `open_in_new_window` function (line 334 and 336), so I think I'm just following the logic before #2903 . And the is_windows flag is already in the `utils.escape_special_chars` function, so not added outside `utils.escape_special_chars`